### PR TITLE
nitrogen8m: Fix BSP override

### PR DESCRIPTION
**Pull request by Ossanes**
Signed-off-by: Vinicius Aquino <vinicius.aquino@ossystems.com.br>

### CODE REVIEW

File: conf/machine/nitrogen8m.conf [hunk 1]
```diff
@@ -64,7 +64,7 @@ IMAGE_BOOT_FILES:append = " \
 
 # wic support
 WKS_FILE = "sdimage-bootpart.wks"
-WKS_FILE_DEPENDS:mx8 += " \
+WKS_FILE_DEPENDS:mx8m-generic-bsp += " \
```
Comment (otavio) [inline conf/machine/nitrogen8m.conf:67]: @GaryThomas please review this PR.

Comment (otavio) [reply conf/machine/nitrogen8m.conf:67]: @gibsson 👆🏼

Comment (otavio) [reply conf/machine/nitrogen8m.conf:67]: I am unsure it will apply or not. @Ossanes please check if dropping it, if it works.

Comment (Ossanes) [reply conf/machine/nitrogen8m.conf:67]: Ok. I'm testing.

**Comment by Ossanes** [comment]
A new PR was open to fix all nitrogen8* machines: #206

**Pull request closed without merging**